.travis.yml

```diff
@@ -30,23 +29,21 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.2'
+    - php: '7.3'
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.3'
       env: SYMFONY=4.4.*
     - php: '7.3'
       env: SYMFONY='dev-master as 4.4.x-dev'
     - php: '7.3'
-      env: SONATA_ADMIN=3.*
-    - php: '7.3'
-      env: SONATA_ADMIN='dev-master as 3.x-dev'
+      env: SONATA_ADMIN='dev-master as 4.x-dev'
     - php: '7.3'
       env: SYMFONY_DEPRECATIONS_HELPER=0
   allow_failures:
     - php: nightly
     - env: SYMFONY_DEPRECATIONS_HELPER=0
     - env: SYMFONY='dev-master as 4.4.x-dev'
-    - env: SONATA_ADMIN='dev-master as 3.x-dev'
```

Comment:
Those changes need to be done on dev-kit too

Reply:
Something like this ? https://github.com/sonata-project/dev-kit/pull/784